src/locale/dim.json

```diff
@@ -491,6 +494,8 @@
     "Store": "Store",
     "Subtitle_Consumable": "{{classType}} {{typeName}}",
     "Subtitle_Gear": "{{light}} {{statName}} {{classType}} {{typeName}}",
+    "Subtitle_Stackable_Unique": "You currently have {{amount}}. You may hold a maximum of {{maxStackSize}}.",
```

Comment:
Hmm, I do worry about the wrapping to two lines, especially in other languages. Could we try making it just something like `Quantity: 45 / 999 (max)` or format it like item stats

Reply:
Agree